<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catMotor</class>
 <widget class="QDialog" name="catMotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asynchronous Motor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="istartLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="rpmLabel">
       <property name="text">
        <string>RPM [1/min]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="efficiencyLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="powerLabel">
       <property name="text">
        <string>Power factor [1]                </string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="loadLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="powerLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="istartLabel">
       <property name="text">
        <string>Istart / Inom [1]</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="rxLabel">
       <property name="text">
        <string>R / X [1]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="rpmLineEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="rxLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="efficiencyLabel">
       <property name="text">
        <string>Efficiency [1]                    </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="loadLabel">
       <property name="text">
        <string>Load factor [1]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="1">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="nominalLabel">
       <property name="text">
        <string>Nominal mech.power[MW]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ratedLabel">
       <property name="text">
        <string>Rated voltage [kV]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="numberLabel">
       <property name="text">
        <string>Number of phases [1]</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ratedVoltageCombo"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="numberofphasesCombo"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nominalLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLabel" name="uidLabel">
       <property name="text">
        <string>UID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="uidLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name                                 </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="aliasLabel">
       <property name="text">
        <string>Alias</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="aliasLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Producer</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="producerComboBox"/>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="descrpitionLabel">
       <property name="text">
        <string>Description                         </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="14" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
